docs/administration/troubleshooting-published-data-packages.md

```diff
@@ -77,23 +77,29 @@ python3 troubleshoot-published-data.py -l
 - The -l option enables logging mode. This saves the log to `/var/lib/irods/log/troubleshoot_publications.log`
 - The -o option enables offline mode, which skips several tests related to connecting to remote servers, but does not skip the datacite test. This is useful when testing on a local development environment.
 - The -n option enables no datacite mode, which skips the datacite checks. This is also useful when testing on a local development environment.
+- The -m option specifies output format: 'human' for readable text (default) or 'csv' for spreadsheet-ready data. 
 
 ## Example output
 
-When checking a single data package, the output containing successful and failed checks displayed in the terminal is as follows:
+When checking a single data package, the output containing successful and failed checks displayed in the terminal is as follows. 
+
+- In human-readable mode:
 
 ```
-Troubleshooting data package: /tempZone/home/vault-core-0/research-core-0[1722266819]
-compare_local_remote_landingpage: File contents at irods path </tempZone/yoda/publication/JCY2C2.html> and remote landing page <https://public.yoda.test/allinone/UU01/JCY2C2.html> do not match.
-Results for: /tempZone/home/vault-core-0/research-core-0[1722266819]
+Troubleshooting Results for: /tempZone/home/vault-default-3/research-default-3[1744029023]
 Package FAILED one or more tests:
-Schema matches: True
-All expected AVUs exist: True
-No unexpected AVUs: True
-Version DOI matches: False
-Base DOI matches: False
-Landing page matches: False
-Combined JSON matches: True
+Schema Check: Pass
+Missing AVUs Check: Pass
+Unexpected AVUs Check: Pass
+Version DOI Check: Fail
+Landing Page Check: Pass
+Combi JSON Check: Pass
+```
+
+- In csv mode:
+
+```
```

Comment:
Can you add the header row here? I think it would help readers understand the meaning of the data row.